mern-ecommerce/routes/order.js

```diff
@@ -29,15 +29,23 @@ router.get(
   requireSignin,
   isAuth,
   isAdmin,
-  getStatusValues
+  getStatus
 );
 
 router.put(
   '/order/:orderId/status/:userId',
   requireSignin,
   isAuth,
   isAdmin,
-  updateOrderStatus
+  updateOrder
+);
+
+router.delete(
+  '/order/:orderId/:userId',
+  requireSignin,
+  isAuth,
+  isAdmin,
+  remove
```

Comment:
The function name in the controller is `exports.removeOrder` but it's being called as `remove` in the route. It should be `removeOrder` instead of `remove`.